resources/docs/TUTORIAL_7_TRAINING_A_MODEL.md

```diff
@@ -316,18 +315,25 @@ trainer.train('resources/taggers/example-ner',
 
 ## Scalability: Training with Large Datasets
 
-Many embeddings in Flair are somewhat costly to produce in terms of runtime and may have large vectors. Examples
-of this are `FlairEmbeddings`, `BertEmbeddings` and the other transformer-based embeddings. Depending on your setup, 
-you can set options to optimize training time. 
+Many embeddings in Flair are somewhat costly to produce in terms of runtime and may have large vectors. 
+Examples of this are `FlairEmbeddings`, `BertEmbeddings` and the other transformer-based embeddings. 
+Depending on your setup, you can set options to optimize training time. 
 
-The main parameter you need to set is the `embeddings_storage_mode` in the `train()` method of the `ModelTrainer`. It can have one of three values:
+The main parameter you need to set is the `embeddings_storage_mode` in the `train()` method of the `ModelTrainer`. 
+It can have one of three values:
 
-1. **'none'**: If you set `embeddings_storage_mode='none'`, embeddings do not get stored in memory. Instead they are generated on-the-fly in each training mini-batch. The main advantage is that this keeps your memory requirements low. However, this 
-also means that embeddings get generated over and over again at each epoch. If you're using computationally costly embeddings such as BERT and you don't have a strong GPU, this may make training very slow. 
+1. **'none'**: If you set `embeddings_storage_mode='none'`, embeddings do not get stored in memory. 
+Instead they are generated on-the-fly in each training mini-batch. 
+The main advantage is that this keeps your memory requirements low.
 
-2. **'cpu'**: If you set `embeddings_storage_mode='cpu'`, embeddings will get stored in regular memory. This in many cases speeds things up significantly since embeddings only need to be computed in the first epoch, after which they are just retrieved from memory. A disadvantage is that this increases memory requirements. Depending on the size of your dataset and your memory setup, this option may not be possible.
+2. **'cpu'**: If you set `embeddings_storage_mode='cpu'`, embeddings will get stored in regular memory. 
```

Comment:
Same as above: During training, it does make sense to have 'cpu' be the default option since this can speed up things a lot. Only during inference (i.e. predict() method) it makes no sense unless you want to use the word embeddings later.

Reply:
ok thanks makes things more clear for me !!!!